docs/code/MlNetCookBook.md

```diff
@@ -779,7 +530,7 @@ IEnumerable<CustomerChurnInfo> churnData = GetChurnInfo();
 // Turn the data into the ML.NET data view.
 // We can use CreateDataView or CreateStreamingDataView, depending on whether 'churnData' is an IList, 
 // or merely an IEnumerable.
-var trainData = mlContext.Data.ReadFromEnumerable(churnData);
+var trainData = mlContext.CreateStreamingDataView(churnData);
```

Comment:
>CreateStreamingDataView [](start = 26, length = 23)

I think now it's `mlContext.Data.ReadFromEnumerable`. We should update the comment as well.